fast/stages/02-networking-vpn/data/firewall-rules/dev/rules.yaml

```diff
@@ -0,0 +1,17 @@
+# skip boilerplate check
```

Comment:
There's no firewall rule for prod.
Also, I'd move stage-specific rules on a separate file.

Reply:
I also just noticed that there are no firewall rules on the `02-networking-nva` stage.
Is it WAI?